tests/macro.rs

```diff
@@ -0,0 +1,17 @@
+use bst_rs::{bst, BinarySearchTree, IterativeBST};
```

Comment:
I think it would be good to refactor the unit tests back into the `*.rs` files in `src`
I started this project when I was very new to Rust and think it's not an ideal structure anymore :D

Feel free to raise another PR refactoring the tests :)